### DISCUSSION TITLE
Service firestore not available

### DISCUSSION THREAD

**Discussion by avibhardwaj233** [vercel/next.js#33798] [Help] [answered]
I am trying to use firestore as my Database with Next.js but whenever I initialize it with `getFirestore()` then it shows me this error:
`Error: Service firestore is not available`

This error never happened before in any other framework like React or Svelte. I also tried creating a new firebase project but it didn't work.
Any help will be greatly appreciated.

**Comment by avibhardwaj233** [comment, marked as answer]
The error is now actually solved, I was just importing `initializeApp()` in the wrong way.

**Reply by djkenneth** [reply]
Hi @avibhardwaj233 how did you solve it? I have the same issue. 

Here is my code.

![code](https://user-images.githubusercontent.com/40983081/209904990-2057eb0d-031d-4226-ae18-4b5e2569fd47.png)

**Reply by asutosh2203** [reply]
I was getting the same error because of wrongly importing initializeApp.
Try this:
`import { initializeApp } from '@firebase/app'`
It worked for me.

**Reply by beelogss** [reply]
@asutosh2203 I have to thank you, you just saved me, bro! thank you so much!! it's only the '@' that's missing from my import.

**Reply by coolnalu** [reply]
Who knew just an `@` could make a huge difference.

**Reply by groovedevelopers** [reply]
its crazy, spent 3 days on this error, only to find out just @ is causing the problem

**Reply by hasebusman** [reply]
You just saved me from having to create a Node.js server! 🙃

**Reply by prajyot-tote** [reply]
Wish I checked it earlier. Almost spent half a day on this. _/|\_
